api-tests/checkout-tests/checkout-api-dev.tests.json

```
@@ -187,9 +187,11 @@
 					"listen": "test",
 					"script": {
 						"exec": [
+							"var jsonData = JSON.parse(responseBody);",
```

For consistency with other tests, prefer:

```suggestion
							"const response = pm.response.json();",
```